packages/flutter/lib/src/widgets/tap_region.dart

```diff
@@ -480,7 +480,7 @@ class TapRegion extends SingleChildRenderObjectWidget {
     return RenderTapRegion(
       registry: TapRegionRegistry.maybeOf(context),
       enabled: enabled,
-      consumeOutsideTaps: consumeOutsideTaps,
+      consumeOutsideTaps: isCurrent && consumeOutsideTaps,
```

Comment:
@gspencergoog Do you know if this is a reasonable solution to TapRegion listening to taps outside of its route?